modules/aggs-matrix-stats/src/main/java/org/elasticsearch/search/aggregations/matrix/stats/InternalMatrixStats.java

```diff
@@ -54,6 +54,7 @@ protected InternalMatrixStats(String name, long count, RunningStats multiFieldSt
      * Read from a stream.
      */
     public InternalMatrixStats(StreamInput in) throws IOException {
+        super(in);
```

Comment:
I'm a bit frightened that the tests didn't fail without this. I expected any serialization problems for this to be caught by the integration tests but I don't think this was because this is only integration tested with REST tests which only uses a single node.